bin/ch-fromhost

```diff
@@ -1,9 +1,19 @@
 #!/bin/sh
-
+#
+# There are three primary uses of ch-fromhost:
+#
+#   1) inject arbitrary files from the host into the container;
+#
+#   2) inject host-built, loadable libfabric (OFI) dynamic shared object (dso)
+#      providers into an image with a MPI built against libfabric to use; and
+#
+#   3) inject nvidia libraries necessary to leverage GPUs with
+#      nvidiia-container-cli.
+#
```

Comment:
Should this go into the documentation rather than a comment?

Reply:
It is, just worded a little differently in the ch_fromhost description.